models/issue_mail.go

```diff
@@ -42,6 +42,15 @@ func mailIssueCommentToParticipants(issue *Issue, doer *User, comment *Comment,
 		participants = append(participants, issue.Poster)
 	}
 
+	// Assignee must receive any communications
+	if issue.AssigneeID != doer.ID {
+		to, err := GetUserByID(issue.AssigneeID)
```

Comment:
What if the assignee has been deleted? `GetUserByID(issue.AssigneeID)` will return an error, but we would want to use a ghost user in this case.

Maybe we should add an `Issue.loadAssignee(..)` method similar to `Issue.loadPoster(..)`. This would also save us from loading the assignee if the assignee has already been loaded.